packages/amplify-console-integration-tests/src/consoleHosting/consoleHosting.ts

```diff
@@ -118,15 +201,7 @@ export function amplifyPush(cwd: string) {
 }
 
 export function npmInstall(cwd: string) {
-  return new Promise((resolve, reject) => {
-    nexpect.spawn('npm install', { cwd, stripColors: true }).run((err: Error) => {
-      if (!err) {
-        resolve();
-      } else {
-        reject(err);
-      }
-    });
-  });
+  spawnSync('npm', ['install'], { cwd });
```

Comment:
this should be moved to e2e-core as we have the same copy in e2e-tests

Reply:
moved